src/webgl/text.js

```diff
@@ -757,6 +759,7 @@ p5.RendererGL.prototype._renderText = function(p, line, x, y, maxY) {
 
     this._doStroke = doStroke;
     this.drawMode = drawMode;
+    gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, true);
```

Comment:
is `gl` defined in this scope?

Reply:
haha you can tell I moved this around -- but it looks like it's already defined on line 684 which is why it didn't break, so I can take out the one I added in this PR!